File: VB/WPFBlankDotNETAppWithNorthwindDatabase/Models/Region.vb
Imports System
Imports System.Collections.Generic

Namespace WPFBlankDotNETAppWithNorthwindDatabase.Models

	Partial Public Class Region
		Public Property RegionId() As Integer

		Public Property RegionDescription() As String

		Public Overridable Property Territories() As ICollection(Of Territory) = New List(Of Territory)()
	End Class
End Namespace

File: CS/WPFBlankDotNETAppWithNorthwindDatabase/Models/Region.cs
using System;
using System.Collections.Generic;

namespace WPFBlankDotNETAppWithNorthwindDatabase.Models;

public partial class Region
{
    public int RegionId { get; set; }

    public string RegionDescription { get; set; }

    public virtual ICollection<Territory> Territories { get; set; } = new List<Territory>();
}
